tutorials/napps/publishing_your_napp.rst

```diff
@@ -70,15 +63,15 @@ using your **username** (the one you have just registered).
    - at least three characters
   --------------------------------------------------------------
 
-The first question is related to your **username**. The second question is the
-name of your NApp, and, for now,
-we will just use ``my_first_napp`` as NApp name. The third question is related
+The first question is related to your **username** (username should be same as your github account username).
+The second question is the name of your NApp, and, for now,
+we will just use ``<name>`` as NApp name. The third question is related
```

Comment:
Krishina, let's update this phrase to to avoid users getting confused with he ``<name>`` as a literal name:

The second question is the name of your NApp  ``<name>``, feel free to use something like ``my_first_napp`` or ``hello_world` for now.